### ISSUE TITLE
Add community guidelines link to footer across all pages

### ISSUE THREAD

**Issue by Tobe01** [DevByte-Community/Community-Website-Frontend#37]
The Community Guidelines page is missing from the footer navigation. Add a link to this page in the footer on all pages to improve accessibility and visibility for users.

**Comment by ossydimma** [comment]
I've done that, the pull request will be submitted soon
